news/2017-07-24-scorewatch-july-week-3.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Scorewatch: July Week 3"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-1
+date: 2017-07-24 15:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-1
+---
+
+A calmer week, but that does not stop a few from pulling off SS plays, impressive osu!taiko and osu!catch plays for this week’s Scorewatch!
```

Comment:
It might've been a bit calmer this week, but that hasn't stopped players from going nuts with scores! From crazy SS plays, to some osu!taiko insanity, and even some fruit-catching madness - we've got it all in this week's issue of Scorewatch!